drivers/src/persistent.rs

```diff
@@ -31,8 +35,11 @@ pub struct PersistentData {
     pub pcr_log: PcrLogArray,
     reserved3: [u8; 4],
 
+    pub measurement_log: StashMeasurementArray,
```

Comment:
We can just put this in the PcrLogArray, right?

Reply:
In case of an update reset, since there are no measurements stashed, resulting in a PCR log that might have stale entries from cold reset flow. Additional logic will be needed to clear the cold-reset PCR log and then start a new one. To keep things simple, I have separated these logs.

Reply:
Sounds good